drivers/hmis/app/jobs/hmis/auto_exit_job.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis
+  class AutoExitJob < BaseJob
+    def perform
+      Hmis::Hud::Project.all.each do |project|
+        config = Hmis::AutoExitConfig.config_for_project(project)
+        next unless config.present?
+
+        project.enrollments.each do |enrollment|
+          most_recent_contact = if HudUtility2024.residential_project_type_ids.include?(project.project_type)
+            enrollment.services.where(record_type: 200).order(:date_provided).last
+          else
+            [
+              enrollment.services.order(:date_provided).last,
+              enrollment.custom_services.order(:date_provided).last,
+              enrollment.current_living_situations.order(:information_date).last,
+              enrollment.custom_assessments.order(:assessment_date).last,
+            ].compact.max_by { |entity| contact_date_for_entity(entity) }
+          end
+
+          next unless most_recent_contact.present?
+
+          most_recent_contact_date = contact_date_for_entity(most_recent_contact)
+          next unless (Date.today - most_recent_contact_date).to_i >= config.length_of_absence_days
```

Comment:
Date.current is better for comparisons https://thoughtbot.com/blog/its-about-time-zones#a-summary-of-do39s-and-don39ts-with-time-zones
```suggestion
          next unless (Date.current - most_recent_contact_date).to_i >= config.length_of_absence_days
```